### ISSUE TITLE
Compatible with transparent bridge mode?

### ISSUE THREAD

**Issue by mrbluecoat** [andybalholm/redwood#81]
Is redwood compatible with Layer 2 transparent bridge mode setups (aka stealth bump-in-the-wire firewall)? If yes, any guidance on a Linux setup?

**Comment by andybalholm** [comment]
No. I think @elico worked on that some, but I don't think he ever got it working. The biggest obstacle is out of the way now, though: it pretty much has a one-to-one correspondence of upstream and downstream connections instead of using connection pools.

**Comment by elico** [comment]
@mrbluecoat depends if you need the origin IP address ie Trpoxy or just Intercept.
The Intercept works out of the box.
The Tproxy support need additional code which requires intensive development and testing.
